src/time_tagger/measurement/dao.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+import TimeTagger as TT
+from shared.constants.constants import INTEGRATION_TIME
+from time_tagger.builder import TimeTaggerBuilder
+
+
+@dataclass
+class CountRateMeasurementReqParams:
+    channels: list[int]
+    time_tagger_network_proxy: object
+
+
+class MeasurementDao:
+    def __init__(self, time_tagger_builder: TimeTaggerBuilder) -> None:
+        self.time_tagger_builder = time_tagger_builder
+
+    def get_count_rates(
```

Comment:
I think the `MeasurementDao` is not really helping much here. For simplicity, I would to suggest moving this method to the `MeasurementService` class.
We can call it `_get_count_rates`, so we mark it as "private"

Reply:
When doing so, we could even remove the CountRateMeasurementReqParams